setup.py

```diff
@@ -19,7 +19,7 @@
     "tqdm~=4.60",
     "plum-dispatch~=1.5.1",
     "numba>=0.52, <0.55",
-    "python-igraph~=0.9",
+    "igraph~=0.9",
```

Comment:
```suggestion
    "igraph~=0.9.8",
```

Reply:
yeah well, no prior versions have been released so...

Reply:
The alternative would be to fix the `conf-compute_oldest_dependecies.py` script, but this looks like a simpler way to get the "oldest dependencies" test working.

Also, note that `~= 1.0` and `~= 1.0.0` [mean different things](https://www.python.org/dev/peps/pep-0440/#compatible-release) in Python version specifiers and I think we want the latter here.